Sources/Subs.php

```diff
@@ -3894,8 +3894,16 @@ function template_javascript($do_deferred = false)
 			}
 
 			else
+			{
 				echo '
-	<script src="', $js_file['fileUrl'], isset($file['options']['seed']) ? $file['options']['seed'] : '', '"', !empty($js_file['options']['async']) ? ' async' : '', !empty($js_file['options']['defer']) ? ' defer' : '', '></script>';
+	<script src="', $js_file['fileUrl'], isset($js_file['options']['seed']) ? $js_file['options']['seed'] : '', '"', !empty($js_file['options']['async']) ? ' async' : '', !empty($js_file['options']['defer']) ? ' defer' : '';
+
+				if (!empty($js_file['options']['attributes']))
+					foreach ($js_file['options']['attributes'] as $key => $value)
+						echo ' ', $key, '="', $value, '"';
```

Comment:
For better HTML5 output, you'll want to handle boolean values differently. This should do the job:

```
foreach ($js_file['options']['attributes'] as $key => $value)
{
	if (is_bool($value))
		echo !empty($value) ? ' ' . $key : '';
	else
		echo ' ', $key, '="', $value, '"';
}
```